src/components/PetAttributesSection.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  Table,
+  TableBody,
+  TableRow,
+  TableCell,
+  TableHeadCell
+} from "@baltimorecounty/dotgov-components";
+import PetAttributes from "./PetAttributes";
+
+const PetAttributesSection = props => {
+  const { attributes, id } = props;
+
+  return (
+    <Table>
+        <TableBody>
+            <TableRow key={id}>
+                <TableHeadCell>Animal ID</TableHeadCell>
+                <TableCell>{id}</TableCell>
+            </TableRow>
+            <PetAttributes attributes={attributes} />
```

Comment:
Not blocking, but we should rename this component in the future. Since this component is embedded in a table it should denote something like `PetAttributeRows` or something along the to let the developer know this component belongs in a table.

Just fyi I removed an extra div in this component to speed up this pr and reformatted, if it looks different.

Reply:
agreed, this name makes more sense